x/bank/app_test.go

```diff
@@ -1,11 +1,13 @@
-package bank
+package bank_test
```

Comment:
Can we rename this file `abci_test.go` and the corresponding file to `abci.go`?

Reply:
this file contains a lot of mixed tests (some from keepers and others from msgs). I'd consider addressing this in a follow PR